src/NuGet.Core/NuGet.Common/XmlUtility.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace NuGet.Common
+{
+    public static class XmlUtility
+    {
+        /// <summary>
+        /// Creates a new System.Xml.Linq.XDocument from an System.Xml.XmlReader.
+        /// </summary>
+        /// <param name="filePath">Path for the file containing the XML data.</param>
+        /// <returns>An <see cref="System.Xml.Linq.XDocument"/> contains the contents of the specified Xml file.</returns>
+        public static XDocument Load(string filePath)
```

Comment:
Validate that `filePath` is not garbage.
```C#
if (string.IsNullOrEmpty(filePath))
{
    throw new ArgumentException(Strings.ArgumentNullOrEmpty, nameof(filePath);
}
```

Reply:
@dtivel - I initially had this null check but removed it because as per the [doc](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmlreader.create?view=netcore-3.1#System_Xml_XmlReader_Create_System_String_), `XmlReader.Create(inputUri)` throws `ArgumentNullExceptio` if `The inputUri value is null`. I will be happy to add this check, if required.

Reply:
Wouldn't it be confusing calling `XmlUtility.Load(filePath: null)` and getting an argument exception for parameter named `inputUri`?